config.yml

```diff
@@ -6,11 +6,5 @@ bots:
   simplebot: screepsbot-zeswarm
 
 serverOptions:
-  # The number of parallel runner threads in which player scripts are executed.
-  # Don't set this option greater than the number of your physical CPU cores.
-  runnerThreads: 4
-  # The number of room processor worker processes to launch.
-  # Don't set this option greater than the number of your physical CPU cores.
-  processorsCnt: 2
```

Comment:
Why is that removed?

Reply:
To not override the defaults specified in `screeps-start.js`, which does the `#cores - 1` for runners, and `#cores` for processors.
People tend to just copy this and leave it, so better to remove it.
I think I will add a section in the wiki for these Server Options though?